File: 17-missing-rust/samples/pi.swift
let n = 355
let d = 113
let whole = n / d
var rem = n % d
var digits = ""
for _ in 0..<4 {
    rem *= 10
    digits += String(rem / d)
    rem %= d
}
print("hello \(whole).\(digits) world!")

File: 17-missing-rust/samples/pi.go
package main

import "fmt"

func main() {
	n, d := 355, 113
	whole, rem := n/d, n%d
	digits := ""
	for i := 0; i < 4; i++ {
		rem *= 10
		digits += fmt.Sprint(rem / d)
		rem %= d
	}
	fmt.Printf("hello %d.%s world!\n", whole, digits)
}
